src/main/java/com/programmers/VoucherManagementApplication/io/Console.java

```diff
@@ -0,0 +1,63 @@
+package com.programmers.VoucherManagementApplication.io;
+
+import com.programmers.VoucherManagementApplication.controller.VoucherController;
+import com.programmers.VoucherManagementApplication.dto.CreateVoucherRequest;
+import com.programmers.VoucherManagementApplication.voucher.Voucher;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class Console implements CommandLineRunner {
+
+    private final Input input;
+    private final Output output;
+    private final VoucherController voucherController;
+
+    public Console(Input input, Output output, VoucherController voucherController) {
+        this.input = input;
+        this.output = output;
+        this.voucherController = voucherController;
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+        MenuType menuType = MenuType.DEFAULT;
+        while (menuType != MenuType.EXIT) {
+            try {
+                output.write(Message.MENU_PROMPT.getMessage());
+                menuType = MenuType.getMenuType(input.readLine());
+                switch (menuType) {
+                    case EXIT:
+                        output.write(Message.EXIT_MESSAGE.getMessage());
+                        break;
+                    case CREATE:
+                        createVoucher();
+                        break;
+                    case LIST:
+                        writeVoucherMap();
+                        break;
+                    default:
+                        output.write(Message.INVALID_INPUT.getMessage());
```

Comment:
```suggestion
	            switch (menuType) {
		            case EXIT -> output.write(Message.EXIT_MESSAGE.getMessage());
		            case LIST -> writeVoucherMap();
		            case CREATE -> createVoucher();
		            default -> output.write(Message.INVALID_INPUT.getMessage());
	            }
```

자바 14부터는 switch Expression 문법으로 이전 보다 가독성이 좋은 방식으로 사용할 수 있습니다.
적용해보시죠!